app/src/main/java/com/example/lastfm/ArtistsData.java

```diff
@@ -0,0 +1,22 @@
+package com.example.lastfm;
+
+public class ArtistsData {
+    private String ArtistName;
+    private String Image;
+    private String PlayCount;
```

Comment:
rename fields according to the code style

Reply:
Changed for proper working with Gson